packages/frontend/src/app/article/[slug]/page.tsx

```diff
@@ -18,6 +18,7 @@ import {
   CMS_URL,
   GetThemeFromCategory,
 } from '@/app/constants'
+import { NewsReading } from './news-reading'
```

Comment:
`page.tsx` was modified in some places, so we may need to update the codebase & fix merge conflicts around here